Sources/Parsers/PlistParser.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  PlistParser.swift
+//  SwiftGenKit
+//
+//  Created by Toshihiro Suzuki on 4/12/17.
+//  Copyright © 2017 SwiftGen. All rights reserved.
+//
+
+import Foundation
+import PathKit
+
+public final class PlistParser {
+
+  var keys: [String] = []
+
+  public init() {}
+
+  public func parse(at path: Path) {
+    guard let keys = loadPlistKeys(at: path) else { return }
+    self.keys = keys
+  }
+
+  fileprivate func loadPlistKeys(at path: Path) -> [String]? {
```

Comment:
Why not make this private? Also: we're in a plist parser, no need to repeat this --> `loadKeys`